addon/package.json

```diff
@@ -40,8 +40,21 @@
     "@babel/core": "^7.16.12",
     "@babel/plugin-proposal-class-properties": "^7.16.7",
     "@babel/plugin-proposal-decorators": "^7.16.7",
+    "@babel/plugin-transform-typescript": "^7.19.3",
+    "@babel/preset-typescript": "^7.18.6",
     "@embroider/addon-dev": "^1.0.0",
```

Comment:
this should be upgraded to `^2.0.0` for typescript support -- it will manage the ts -> js transformation much better, and use correct entries in the `app-js` below

Reply:
Fixed